### PR TITLE
Fix batch norm multiplier init

### LINKED ISSUE #12259: Set Batchnorm weight scalar initialization to unit (not random uniform)
## 🐛 Feature suggestion

Pytorch Batch normalization currently initializes it's scalar weight parameter with random uniform variables (https://github.com/pytorch/pytorch/blob/1d3f650ce4f1b781f03e8b4f250a25d5a8f819cc/torch/nn/modules/batchnorm.py#L46). This initialization has no theoretical basis, while on the other hand it makes sense to use a fixed value of 1. 

Part of the usefulness of Batchnorm is that it can initialize the network easily, with inputs to hidden units normalized to zero norm and unit variance (original paper: https://arxiv.org/abs/1502.03167) and it is also how recent studies use it (e.g. https://arxiv.org/abs/1706.02677).

## To change

From 
init.uniform_(self.weight)

to 
init.ones_(self.weight)

### PR DESCRIPTION
**Pull request by Kaixhin**
Fixes #12259

### CODE REVIEW

**Comment by colesbury** [comment]
I think this is a good change. We have been manually initializing the weight to `1` in models like [ResNet](https://github.com/pytorch/vision/blob/86001a871d3335046e2dca7715d9babf73e6956f/torchvision/models/resnet.py#L116). It's silly to require users to remember to do that.

cc @soumith

**Review by soumith — APPROVED** [review]

**Comment by soumith** [comment]
sounds good to me. let's land it
